### ISSUE TITLE
Add Escape key handler for modal dismissal for editing a resource

### ISSUE THREAD

**Issue by nro337** [nro337/nexus#15]
The modal supports backdrop click to close but lacks keyboard support for dismissal. Users should be able to press Escape to close the modal, which is a standard accessibility pattern.

https://github.com/nro337/nexus/pull/12/changes#r2929294149
